Ginger/Ginger/ALM/ALMDefectsProfilesPage.xaml.cs

```diff
@@ -346,5 +341,26 @@ public static ALMDefectProfile SetALMDefectProfileWithDefaultValues(string name
 
             return newALMDefectProfile;
         }
+
+        ObservableList<ExternalItemFieldBase> FetchDefectFields(GingerCoreNET.ALMLib.ALMIntegration.eALMType AlmType)
+        {
+            ObservableList<ExternalItemFieldBase> defectFields = new ObservableList<ExternalItemFieldBase>();
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'defectFields'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5717441)